R/bayesian/nodesplit/nodesplit_panel.R

```diff
@@ -47,19 +54,38 @@ nodesplit_panel_server <- function(
         )
       }
     )
-
-    output$node_table<- renderTable(
-      colnames = TRUE,
-      {
-        model_nodesplit()
-      }
+  
+    
+    output$node_plot_placeholder <- renderUI({
+      plotOutput(
+        outputId = ns("node_plot"),
+        height = NodePixels(
+          as.numeric(length(model_nodesplit())-1)
+        )
+      )
+    })
+    
+    output$node_plot<- renderPlot({
+      req(input$node)
+      plot(summary(model_nodesplit()), digits = 3)
+    }
     )
 
     output$downloadnode <- downloadHandler(
-      filename = 'Nodesplit.csv',
+      filename = function() {
+        paste0("Nodesplit.",
+               input$download_format)
+      },
       content = function(file) {
-        write.csv(model_nodesplit(), file)
-      }
+        if (input$download_format == "PDF") {
+          pdf(file = file, width = 9)
+        } else if (input$download_format == "PNG") {
+          png(file = file, width = 9)
```

Comment:
The png download doesn't work, which I believe is because the default units for the width is pixels, and 9 isn't big enough.